_docs/reference/cli/cluster.md

```diff
@@ -5,4 +5,53 @@ anchor: reference
 module: reference/cli/cluster
 ---
 
-{% include wip.md %}
+# Cluster
```

Comment:
Maybe hide this until 0.8?  Probably need to make it clear that this will not actually create a cluster, but it will provision the discovery service for a new one.